docs/learn/learn-balance-transfers.md

```diff
@@ -9,9 +9,10 @@ slug: ../learn-balance-transfers
 Balance transfers are used to send a balance from one account to another account. To start
 transferring a balance, we will begin by using [Polkadot-JS Apps][]. This guide assumes that you've
 already [created an account](learn-account-generation.md) and have some funds that are ready to be
-transferred.
+transferred. This guide also contains instructions on how to sign the balance transfer transactions
+for accounts created on Polkadot JS Extension, Ledger devices and Parity Signer App.
 
-## Polkadot-JS Apps
+## Polkadot-JS Apps UI
```

Comment:
I believe we can refer to Polkadot-JS Apps as just "Polkadot-JS Apps" or "Apps UI"

Reply:
Yeah, whichever one that keeps uniformity across the Wiki and Support pages. Started discussion in Tech Ed and Community channel.